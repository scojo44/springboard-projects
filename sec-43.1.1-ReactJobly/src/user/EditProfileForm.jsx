import React from 'react'
// import './EditProfileForm.css'

export default function EditProfileForm() {
  return (
    <>
    EditProfileForm
    </>
  );
}
