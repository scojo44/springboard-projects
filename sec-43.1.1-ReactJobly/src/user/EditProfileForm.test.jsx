import {render} from '@testing-library/react'
import {MemoryRouter} from 'react-router-dom'
import EditProfileForm from './EditProfileForm';

describe('EditProfileForm Tests', () => {
  it('Renders without crashing', () => {
    render(
      <MemoryRouter>
        <EditProfileForm />
      </MemoryRouter>
    );
  });

  it('Matches snapshot', () => {
    const {asFragment} = render(
      <MemoryRouter>
        <EditProfileForm />
      </MemoryRouter>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
