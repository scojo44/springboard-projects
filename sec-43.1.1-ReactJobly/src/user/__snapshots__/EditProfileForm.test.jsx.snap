// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`EditProfileForm Tests > Matches snapshot 1`] = `
<DocumentFragment>
  EditProfileForm
</DocumentFragment>
`;
